support/Pipelines/Bbh/Inspiral.py

```diff
@@ -35,7 +93,45 @@ def inspiral_parameters(
     """
     id_domain_creator = id_input_file["DomainCreator"]["BinaryCompactObject"]
     id_binary = id_input_file["Background"]["Binary"]
-    return {
+    left_object = id_binary["ObjectLeft"]
+    right_object = id_binary["ObjectRight"]
+
+    assert len(left_object) == 1, "Expected only one option for 'ObjectLeft'"
+    assert len(right_object) == 1, "Expected only one option for 'ObjectRight'"
+
+    left_object_type, left_object_params = list(left_object.items())[0]
+    right_object_type, right_object_params = list(right_object.items())[0]
+
+    supported_solutions = [
+        "KerrSchild",
+        "HarmonicSchwarzschild",
+        "Schwarzschild",
+        "SphericalKerrSchild",
+    ]
+    assert left_object_type in supported_solutions, (
+        f"Unknown solution for 'ObjectLeft': '{left_object_type}'. Supported"
+        f" solutions are: {supported_solutions}"
+    )
+    assert right_object_type in supported_solutions, (
+        f"Unknown solution for 'ObjectRight': '{right_object_type}'. Supported"
+        f" solutions are: {supported_solutions}"
+    )
+
+    # ID parameters
+    mass_left = left_object_params["Mass"]
+    mass_right = right_object_params["Mass"]
```

Comment:
These masses are incorrect. They are the conformal masses for the Xcts solve. We actually need the Christodoulou/ADM masses for this. We can only get those from horizon finds after the ID solve.

I've been testing some runs with these (incorrect) masses and they don't matter too much for the control system params, but they are bad for the constraint damping params. My bbh runs die quickly. So I think we should hold off on this temporarily until we have horizon finds after the ID solve. Either I or @nilsvu probably need to do that.

Reply:
Yea I'll get you the horizon masses, though for equal-mass nonspinning you know they are ~0.5. Does the ~20% difference to the conformal masses matter so much for the constraint damping? Does the constraint damping work if you set the masses to 0.5?

Reply:
Yes it matters quite a bit for the constraint damping parameters. Using the correct masses of 0.5 allowed the simulation to run much longer than with the incorrect masses.

Reply:
I agree with @knelli2 that you need to set the black-hole masses from the results of a horizon find, not the conformal masses

Reply:
I have a branch do get horizon quantities from initial data here: https://github.com/nilsvu/spectre/commits/id_horizon_quantities/

You can call `find_horizon` on the initial data here in this function to get the quantities. We can also store these alongside the initial data later, maybe in the reductions file.

Reply:
Here's the PR to get the horizon masses and spins: https://github.com/sxs-collaboration/spectre/pull/6024

Reply:
Once #6024 is merged, I'll use that here to get the correct masses and spins

Reply:
#6024 is merged now, so you can go ahead with this 👍